core/templates/dev/head/components/skill-mastery/skill-mastery.constants.ts

```diff
@@ -0,0 +1,21 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Constants for the skill mastery viewer.
+ */
+
+export class SkillMasteryConstants {
+  public static GOOD_MASTERY_CUTOFF = 0.7;
```

Comment:
Is this part of the milestone 3.2 change?

Reply:
As discussed in our meeting, will change it once 3.2 PR goes in.

Reply:
Done.